Tests/AK/TestSlugify.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <AK/Slugify.h>
+
```

Comment:
General comment: Add some tests using non-ascii characters to make sure that works, once that's supported.

Reply:
Yes added.